packages/eslint-plugin-js/rules/comma-dangle/comma-dangle.ts

```diff
@@ -237,14 +247,14 @@ export default {
             yield fixer.remove(trailingToken)
 
             /*
```

Comment:
```suggestion
            /**
```

ESLint will auto format with this